sdk/eventgrid/perf-tests/eventgrid/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@azure-tests/perf-eventgrid",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@azure/eventgrid": "^3.0.0-beta.3",
+    "@azure/test-utils-perfstress": "^1.0.0",
+    "dotenv": "^8.2.0"
+  },
+  "devDependencies": {
+    "@types/dotenv": "8.2.0",
```

Comment:
`"@types/dotenv"` is deprecated.. 

https://www.npmjs.com/package/@types/dotenv

Latest "dotenv" provides the types.
```suggestion
```